### DISCUSSION TITLE
How to get datasource property from another extension ?

### DISCUSSION THREAD

**Discussion by scandinave** [quarkusio/quarkus#41192] [Q&A] [answered]
Hi, 
I develop an extension for a web service so that this web service can be easily integrated and started in dev mode into others applications. I use testContainer to launch the web service when the application that depends on it start.

This web service depends on a database, so I need the JDBC url, username, password to passe it to the web service container at start. I have use a `DevServicesDatasourceResultBuildItem` that give me a username and password config and also JDBC url if specified in the application.properties of the main application. Bu I want full auto configuration where the application does not specified database url. 

In this case, `DevServicesDatasourceResultBuildItem` does not contains JDBC Url anymore. How can I fix that? 

thx

**Comment by scandinave** [comment, marked as answer]
HI. I found that my approach is not the best. DevService need to be isolated from each other. So i should not use the same database spawn by the main application. This pose the problem of using the same database for my devservice and the main app. 

So my solution is to use by extension to spawn two containers One for the devservice database and the other for the devservice itself.

**Reply by ZetalyAlexandre** [reply]
Hello @scandinave,
I'm having the same problem with the need to launch another Dev Service container (database or other application) from the Dev Service I'm creating. However, I can't find any documentation on how to do it. By any chance, have you found any literature on this subject?
